packages/admin/cms-admin/src/generator/future/generateForm/generateFields.ts

```diff
@@ -10,6 +10,12 @@ export type GenerateFieldsReturn = GeneratorReturn & {
     hooksCode: string;
     formFragmentFields: string[];
     formValueToGqlInputCode: string;
+    formValuesConfig: {
+        omitFromFragmentType?: string;
+        typeCode: string[];
+        initializationCode: string[];
+        defaultInitializationCode: string[];
```

Comment:
What's the benefit of using an array here, and not a string? You could join the lines here as well